diff --git a/configure b/configure
index 281f111111..3243e23021 100644
--- a/configure
+++ b/configure
@@ -6189,11 +6189,11 @@ case $target_os in
     win32|win64)
         disable symver
         LIBSUF=".lib"
-        if enabled shared; then
+        # if enabled shared; then
             # Cannot build both shared and static libs with MSVC or icl.
-            disable static
-            LIBSUF="-static.lib"
-        fi
+        #    disable static
+        #    LIBSUF="-static.lib"
+        # fi
         ! enabled small && test_cmd $windres --version && enable gnu_windres
         enabled x86_32 && check_ldflags -LARGEADDRESSAWARE
         add_cppflags -DWIN32_LEAN_AND_MEAN
